01-import_and_maxfilter.py

```diff
@@ -44,7 +43,8 @@
 from mne.parallel import parallel_func
 from mne_bids import make_bids_basename, read_raw_bids, get_matched_empty_room
 from mne_bids.config import BIDS_VERSION
-from mne_bids.utils import _write_json, _parse_bids_filename
+from mne_bids.utils import _write_json
+from mne_bids.path import _parse_bids_filename
```

Comment:
do any of these functions need to be made public?

Reply:
We should consider making `_parse_bids_filename` public, yes